.pre-commit-config.yaml

```
@@ -92,3 +92,9 @@ repos:
       ^[^/]+[.]rst$
     exclude: >-
       ^CHANGES\.rst$
+- repo: https://github.com/codespell-project/codespell
+  rev: v2.2.6
+  hooks:
+  - id: codespell
+    additional_dependencies:
+      - tomli
```

(the file uses zero-indent YAML sequences)
```suggestion
    - tomli
```